### PR TITLE
New axol Finality Provider

### PR DESCRIPTION
**Pull request by Hydepwns**
# New axol Finality Provider

## Checklist

- [x] I have followed the finality provider information registry
[guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-1/finality-providers/README.md)
- [x] I have backed up my mnemonic
- [x] I have read and agree to the [Babylon Ecosystem Participant License](https://docs.babylonlabs.io/assets/files/babylon-ecosystem-participant-license.pdf) and the [Babylon Ecosystem Participant Agreement](https://docs.babylonlabs.io/assets/files/babylon-ecosystem-participant-agreement.pdf).

> [!CAUTION]
> The loss of the (generated keys + mnemonic) makes the finality provider
useless and unable to provide finality, which would lead to no transition to
later phases of the Babylon network.

### CODE REVIEW

**Comment by RafilxTenfen** [comment]
Hey @Hydepwns can you update the checkbox to be [x] or set as [ ] and then click on it after?

**Comment by Hydepwns** [comment]
done @RafilxTenfen ! lfg

File: bbn-1/finality-providers/registry/axol.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": {
+    "moniker": "axol.io",
+    "identity": "FA1EA630E6129D0C",
+    "website": "https://axol.io/",
+    "security_contact": "drew@axol.io",
+    "details": "Axol is empowering institutional staking infrastructure with unrivaled uptime, harnessing self-healing nodes and renewable energy across Latin America. Contact us to stake your bitcoin today."
+  },
+  "eots_pk": "311c550309edab2025b9ee907cfeb0cf7223fefdda5d7752f2abe765d34d11cd",
+  "commission": "0.01"
```
Comment (RafilxTenfen) [inline bbn-1/finality-providers/registry/axol.json:10]: min commission is 3% 0.03, can you update and resign?

Comment (RafilxTenfen) [reply bbn-1/finality-providers/registry/axol.json:10]: @Hydepwns

**Review by RafilxTenfen — CHANGES_REQUESTED** [review]

File: bbn-1/finality-providers/registry/axol.json (hunk 1 above)
Comment (Hydepwns) [reply bbn-1/finality-providers/registry/axol.json]: done @RafilxTenfen !!

**Comment by Hydepwns** [comment]
I have also resigned these changes and updated signatures

**Review by RafilxTenfen — APPROVED** [review]
:tada:

**Review by vitsalis — APPROVED** [review]

**Pull request merged**